.travis.yml

```diff
@@ -1,29 +1,29 @@
 language: go
 
 go:
-  - 1.5
```

Comment:
IIRC, this was mainly done to reduce the build time on travis, as the method I put in place in this PR builds both ffmpeg  3.0, 3.3 it took like 1-2 hours.
I'll try to re-enable them!